services/Facebook.json

```diff
@@ -23,6 +23,23 @@
         "removeHelpButtons",
         "cleanUrls"
       ]
+    },
+    "cookiesPolicy": {
+      "location": "https://www.facebook.com/policies/cookies",
+      "contentSelector": "._5tkp",
+      "filters": [
+        "removeImgFromCookiesPolicy",
+        "removeReturnToTopButtons",
```

Comment:
I still see an empty link, dunno if it should be removed as a “return to top”… ?

![Capture d’écran 2020-08-05 à 16 41 02](https://user-images.githubusercontent.com/222463/89426440-73da9000-d73a-11ea-81fc-23c1afb0ada3.png)

Reply:
I modified function cleanURLs to remove empty links.